lib/common/theme.dart

```diff
@@ -211,6 +211,9 @@ class ProvenanceThemeData {
   static const Color _notice350 = Color(0xFFF4B601);
   static const Color _notice800 = Color(0xFF140F00);
   static const Color _positive300 = Color(0xFF5AE70D);
+  static const Color _depositPeriod = Color(0xFFFEC10B);
```

Comment:
These new names are context-specific, whereas the others are generic. Should these new names be generic?